oled.js

```diff
@@ -1,743 +1,740 @@
-var Oled = function(board, five, opts) {
-
-  this.HEIGHT = opts.height || 32;
-  this.WIDTH = opts.width || 128;
-  this.ADDRESS = opts.address || 0x3C;
-  this.PROTOCOL = (opts.address) ? 'I2C' : 'SPI';
-  this.MICROVIEW = opts.microview || false;
-  this.SLAVEPIN = opts.slavePin || 12;
-  this.RESETPIN = opts.resetPin || 4;
-  this.DATA = opts.data || 0x40;
-  this.COMMAND = opts.command || 0x00;
-
-  // create command buffers
-  this.DISPLAY_OFF = 0xAE;
-  this.DISPLAY_ON = 0xAF;
-  this.SET_DISPLAY_CLOCK_DIV = 0xD5;
-  this.SET_MULTIPLEX = 0xA8;
-  this.SET_DISPLAY_OFFSET = 0xD3;
-  this.SET_START_LINE = 0x00;
-  this.CHARGE_PUMP = 0x8D;
-  this.EXTERNAL_VCC = false;
-  this.MEMORY_MODE = 0x20;
-  this.SEG_REMAP = 0xA1; // using 0xA0 will flip screen
-  this.COM_SCAN_DEC = 0xC8;
-  this.COM_SCAN_INC = 0xC0;
-  this.SET_COM_PINS = 0xDA;
-  this.SET_CONTRAST = 0x81;
-  this.SET_PRECHARGE = 0xd9;
-  this.SET_VCOM_DETECT = 0xDB;
-  this.DISPLAY_ALL_ON_RESUME = 0xA4;
-  this.NORMAL_DISPLAY = 0xA6;
-  this.COLUMN_ADDR = 0x21;
-  this.PAGE_ADDR = 0x22;
-  this.INVERT_DISPLAY = 0xA7;
-  this.ACTIVATE_SCROLL = 0x2F;
-  this.DEACTIVATE_SCROLL = 0x2E;
-  this.SET_VERTICAL_SCROLL_AREA = 0xA3;
-  this.RIGHT_HORIZONTAL_SCROLL = 0x26;
-  this.LEFT_HORIZONTAL_SCROLL = 0x27;
-  this.VERTICAL_AND_RIGHT_HORIZONTAL_SCROLL = 0x29;
-  this.VERTICAL_AND_LEFT_HORIZONTAL_SCROLL = 0x2A;
-
-  this.cursor_x = 0;
-  this.cursor_y = 0;
-
-  // new blank buffer
-  this.buffer = new Buffer((this.WIDTH * this.HEIGHT) / 8);
-  this.buffer.fill(0x00);
-
-  this.dirtyBytes = [];
-
-  // this is necessary as we're not natively sitting within johnny-five lib
-  this.board = board;
-  this.five = five;
```

Comment:
this line, and the line above it, are necessary for the library to work properly. When I tested this manually with an OLED screen, I stumbled upon these missing LOC in your branch. Could you please commit them back in? Thanks!

Reply:
I'm sorry I must have missed those. I added them back.